wireless_modules/mqtt_test/boot.py

```diff
@@ -0,0 +1,27 @@
+from config import ESSID
+from config import PASSWORD
+
+def connect(eesid, password):
+    import network
+    sta_if = network.WLAN(network.STA_IF)
+    if not sta_if.isconnected():
+        print('connecting to network...')
+        sta_if.active(True)
+        sta_if.connect(eesid, password)
+        while not sta_if.isconnected():
+            pass
+
+    print('network config:', sta_if.ifconfig())
+
+def no_debug():
```

Comment:
What does this function do?

Reply:
It allows for the code to be uploaded and run. If you don't disable it, you may not be able to use ampy or rshell to upload your code to the board

Reply:
I'll add more comments to the scripts as it is a bit confusing